example/use_cases/apartment_application/apartment_application.go

```diff
@@ -0,0 +1,150 @@
+
+package main
+
+import (
+	"crypto/ed25519"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/goccy/go-json"
+	"github.com/stretchr/testify/assert"
+)
+
+func main() {
+
+	/**
+		 Step 1: Create new entities as DIDs. Govt Issuer, User Holder, and Apartment Verifier
+	**/
+
+	// User Holder
+	holderDIDPrivateKey, holderDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	holderJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(holderDIDPrivateKey.(ed25519.PrivateKey))
+	holderSigner, _ := cryptosuite.NewJSONWebKeySigner(holderJWK.ID, holderJWK.PrivateKeyJWK, cryptosuite.Authentication)
+	holderVerifier, _ := cryptosuite.NewJSONWebKeyVerifier(holderJWK.ID, holderJWK.PublicKeyJWK)
+
+	// Apt Verifier
+	aptDidPrivateKey, aptDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	aptJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(aptDidPrivateKey.(ed25519.PrivateKey))
+	aptSigner, _ := cryptosuite.NewJSONWebKeySigner(aptJWK.ID, aptJWK.PrivateKeyJWK, cryptosuite.Authentication)
+	aptVerifier, _ := cryptosuite.NewJSONWebKeyVerifier(aptJWK.ID, aptJWK.PublicKeyJWK)
+
+	// Government Issuer
+	govtDidPrivateKey, govtDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
+	govtJWK, _ := cryptosuite.JSONWebKey2020FromEd25519(govtDidPrivateKey.(ed25519.PrivateKey))
+	govtSigner, _ := cryptosuite.NewJSONWebKeySigner(govtJWK.ID, govtJWK.PrivateKeyJWK, cryptosuite.Authentication)
+
+	fmt.Print("\n\nStep 1: Create new DIDs for entities\n\n")
+	fmt.Printf("Tenant: %s\n", string(*holderDIDKey))
+	fmt.Printf("Apartment: %s\n", string(*aptDIDKey))
+	fmt.Printf("Government: %s\n", string(*govtDIDKey))
+
+	/**
+		 Step 2: Government issuer issues credentials to holder claiming age. The government issuer then signs the verifiable credentials to holder claiming age
+	**/
+
+	knownIssuer := govtDIDKey
+	knownIssuanceDate := "2020-01-01T19:23:24Z"
+	knownSubject := map[string]interface{}{
+		"id":        string(*holderDIDKey),
+		"birthdate": "1975-01-01",
+	}
+
+	vcBuilder := credential.NewVerifiableCredentialBuilder()
+
+	vcBuilder.SetIssuer(string(*knownIssuer))
+	vcBuilder.SetIssuanceDate(knownIssuanceDate)
+	vcBuilder.SetCredentialSubject(knownSubject)
+
+	vc, _ := vcBuilder.Build()
+	assert.NoError(nil, vc.IsValid())
+
+	signedVCBytes, _ := signing.SignVerifiableCredentialJWT(*govtSigner, *vc)
+
+	fmt.Print("\n\nStep 2: Government issues Verifiable Credential new for tenant verifying birthdate and signs\n\n")
+	if dat, err := util.PrettyJSON(vc); err == nil {
+		fmt.Printf("Verifiable Credential:%s\n", string(dat))
+	}
+
+	/**
+		Step 3: Create presentation definition from the apartment to the holder which goes into a presentation request.
```

Comment:
maybe language fix to make it clear that the physical apartment building is not making the request 😉